katalog/fluentd/fluentbit.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentbit
+  namespace: logging
+  labels:
+    app: fluentbit
+spec:
+  selector:
+    matchLabels:
+      app: fluentbit
+  template:
+    metadata:
+      labels:
+        app: fluentbit
+    spec:
+      tolerations:
+        - operator: Exists
+      containers:
+        - image: fluent/fluent-bit:1.4.4
```

Comment:
Try yo put the image tag in the `kustomization.yaml` file please ;)

Reply:
Will do!